src/CoreBundle/Resources/public/components/select2.v4/js/i18n/af.js

```diff
@@ -0,0 +1,3 @@
+/*! Select2 4.0.6-rc.1 | https://github.com/select2/select2/blob/master/LICENSE.md */
```

Comment:
Yes, we need this one, because otherwise a "clear" event handling is missing. Tried the 4.0.5. The 4.0.6.RC1 is one year old. I really hope they will release a final release shortly.

Reply:
This issue doesn't sound that good: https://github.com/select2/select2/issues/5418

Are there still other libraries we could use that are actively maintained (I'd suggest helping in maintaing that lib, but those 355 open issues and 100 open PRs are intimidating)?

Reply:
damned... this is bad, but maybe fixing the issues that are a problem for us (and at the moment it works) will be the right way. I checked for some alternatives. This one is proposed, but last official release is from 2016 (last commit is 6 months old, so there is something going on)! https://github.com/selectize/selectize.js/releases 
I would not suggest to replace select2 at this point. We had a lot of afford to make it work and now it works quite well.